src/store/modules/user/actions.ts

```diff
@@ -78,8 +78,15 @@ const actions: ActionTree<UserState, RootState> = {
     },
 
   // update current facility information
-  async setFacility ({ commit }, payload) {
+   async setFacility ({ commit }, payload) {
     commit(types.USER_CURRENT_FACILITY_UPDATED, payload.facility);
+  },
+
+  /**
+   * Set User Instance Url
+   */
+   setUserInstanceUrl ({ state, commit }, payload){
```

Comment:
Improve Indentation

Reply:
Improved Indentation.
Thank You sir for reviewing my code.